projects/ngx-charts-on-fhir/src/lib/fhir-chart-summary/fhir-chart-summary-card/fhir-chart-summary-card.component.html

```diff
@@ -1,11 +1,6 @@
-<mat-card class="summary-card" [style.background]="getBackgroundStyle()">
-  <mat-card-header>
-    <mat-card-title-group>
-      <mat-card-subtitle *ngIf="subtitle">{{ subtitle }}</mat-card-subtitle>
-      <mat-card-title *ngIf="title" [matTooltip]="title">{{ title }}</mat-card-title>
-    </mat-card-title-group>
-  </mat-card-header>
-  <mat-card-content>
+<div class="card" class="summary-card" [style.background]="getBackgroundStyle()">
+  <div class="card-header"></div>
+  <div class="card-body">
     <ng-content></ng-content>
-  </mat-card-content>
-</mat-card>
\ No newline at end of file
+  </div>
+</div>
```

Comment:
are all these divs necessary? it looks like only `summary-card` is applying styles. also, please cleanup unused rules from the css file.

Reply:
also that outer div has two `class` attributes